### PR TITLE
Create and connect DB for users. Add profile page

### PR DESCRIPTION
**Pull request by tkachuktaras**

### CODE REVIEW

File: blocks/head.php [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <meta charset="utf-8">
-<link rel="icon" href="./images/ico.png" type="image/png">
+<link rel="icon" href="../images/ico.png" type="image/png">
```
Comment (yurykozhenov) [inline blocks/head.php]: Use absolute path

File: blocks/articles.php [hunk 2]
```diff
@@ -7,7 +7,7 @@
         <div class="article__maintext">
             <?php echo $row["text"]?>
         </div>
-        <a href="./article.php?id=<?php echo $row["id"]?>" class="link__arcticle">
+        <a href="/pages/article.php?id=<?php echo $row["id"]?>" class="link__arcticle">
```
Comment (yurykozhenov) [inline blocks/articles.php]: `link__arcticle` -> `link__article`

File: blocks/head.php [hunk 3]
```diff
@@ -1,6 +1,6 @@
...
 <script src="https://kit.fontawesome.com/46e75e08cb.js" crossorigin="anonymous"></script>
 <link rel="preconnect" href="https://fonts.gstatic.com">
 <link href="https://fonts.googleapis.com/css2?family=Comfortaa:wght@300;400;500;600;700&display=swap" rel="stylesheet">
-<link rel="stylesheet" href="css/style.css">
\ No newline at end of file
+<link rel="stylesheet" href="../css/style.css">
```
Comment (yurykozhenov) [inline blocks/head.php]: Use absolute path

File: pages/change_profile.php [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+                                <tr>
+                                    <td class="tittle">День рождения</td>
+                                    <td><input name="birthday" id="birthday" value="<?php echo $user['birthday']?>" class = "input_profile" placeholder="День Рождения"></td>
+                                </tr>
+                                <tr>
+                                    <td class="tittle">Страна</td>
+                                    <td><input name="country" id="country" value="<?php echo $user['country']?>" class = "input_profile" placeholder="Страна"></td>
+                                </tr>
+                                <tr class="bottom__border">
+                                    <td class="tittle">Город</td>
```
Comment (ghost) [inline pages/change_profile.php]: change "tittle"

File: css/style.css [hunk 5]
```diff
@@ -143,6 +143,14 @@ h6 {
     box-shadow: -3px 6px 9px #061e3756;
 }
 
+.backgound__opasity{
```
Comment (yurykozhenov) [inline css/style.css]: `opasity` -> `opacity`

File: pages/article.php [hunk 6]
```diff
@@ -9,13 +8,13 @@
 <html>
 
 <head>
-    <?php require "./blocks/head.php"; ?>
+    <?php require "../blocks/head.php" ?>
     <title><?php echo $row["tittle"]?></title>
-    <link rel="stylesheet" href="./css/article.css">
+    <link rel="stylesheet" href="../css/article.css">
```
Comment (yurykozhenov) [inline pages/article.php]: Use absolute path

File: pages/article.php [hunk 7]
```diff
@@ -51,7 +50,7 @@
             <div class="bottom__part">
                 <div class="write__comments">
                     <div class="upper">
-                        <img src="./images/user.png" class="user__img">
+                        <img src="../images/user.png" class="user__img">
```
Comment (yurykozhenov) [inline pages/article.php]: Use absolute path for all images

File: pages/submit_changes.php [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+                            exit();
+                        }
+                        else{
+                            if(mb_strlen($country) <= 4 || mb_strlen($country) >= 20){
+                                echo "<h1>Введите страну правильно</h1>";
+                                exit();
+                            }
+                            else{
+                                if(mb_strlen($city) <= 2 || mb_strlen($city) >= 40){
+                                    echo "<h1>Введите страну правильно</h1>";
```
Comment (ghost) [inline pages/submit_changes.php]: maybe here you mean "город"?

File: pages/change_profile.php [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+</head>
+
+<body>
+    <?php require "../blocks/header.php" ?>
+        <div class="container">
+            <form action="submit_changes.php?id=<?php echo $user["id"]?>" method="post">
+                <div class="center">
+                    <div class="profile">
+                        <div class="left__profile">
+                            <div class="tittle__profile">
```
Comment (yurykozhenov) [inline pages/change_profile.php]: `tittle__profile` -> `title__profile`

File: pages/change_profile.php [hunk 10]
```diff
@@ -0,0 +1,83 @@
...
+                                <tr>
+                                    <td class="tittle">Логин</td>
+                                    <td><input name="login" id="login" class = "input_profile" value="<?php echo $user['login']?>" placeholder="Логин"></td>
+                                </tr>
+                                <tr>
+                                    <td class="tittle">Почта</td>
+                                    <td><input name="email" id="email" class = "input_profile" value="<?php echo $user['email']?>" placeholder="Почта"></td>
+                                </tr>
+                                <tr>
+                                    <td class="tittle">Имя и фамилия</td>
```
Comment (yurykozhenov) [inline pages/change_profile.php]: `tittle` -> `title`

File: pages/login.php [hunk 11]
```diff
@@ -2,25 +2,25 @@
 <html>
 
 <head>
-    <?php require "./blocks/head.php"; ?>
+    <?php require "../blocks/head.php"; ?>
     <title>Авторизация</title>
-    <link rel="stylesheet" href="css/login_page.css">
+    <link rel="stylesheet" href="../css/login_page.css">
```
Comment (yurykozhenov) [inline pages/login.php]: Use absolute path

File: pages/submit_changes.php (hunk 8 above)
Comment (tkachuktaras) [reply pages/submit_changes.php]: yes sir

> maybe here you mean "город"?

File: pages/submit_changes.php [hunk 12]
```diff
@@ -0,0 +1,87 @@
...
+    if($_COOKIE['usertime'] !== NULL){
+        if($_COOKIE['usertime'] != $user['login']){
+            header('Location: /');
+            exit();
+        }
+    }
+    else{
+        header('Location: /');
+        exit();
+    }
```
Comment (yurykozhenov) [inline pages/submit_changes.php]:
```suggestion
    if($_COOKIE['usertime'] === NULL || $_COOKIE['usertime'] !== $user['login']) {
        header('Location: /');
        exit();
    }
```

File: pages/submit_changes.php [hunk 13]
```diff
@@ -0,0 +1,87 @@
...
+    $country = filter_var(trim($_POST['country']),FILTER_SANITIZE_STRING);
+
+    $city = filter_var(trim($_POST['city']),FILTER_SANITIZE_STRING);
+
+
+    if(mb_strlen($login) <= 5 || mb_strlen($login) >= 25){
+            echo "<h1>Логин слишком маленький</h1>";
+            exit();
+        }
+    else{
```
Comment (yurykozhenov) [inline pages/submit_changes.php]: Remove `else`. You do not need `else` after `exit()`

File: pages/submit_changes.php [hunk 14]
```diff
@@ -0,0 +1,87 @@
...
+                                    exit();
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+    mysqli_connect("localhost", "root", "", "stalker");
```
Comment (yurykozhenov) [inline pages/submit_changes.php]: Remove connection, it is already created in `connect_to_db.php`

File: pages/check.php [hunk 15]
```diff
@@ -0,0 +1,85 @@
...
+    require "../blocks/connect_to_db.php";
+
+    $result = $mysql->query("SELECT * FROM `users` WHERE `login`= '$login' AND `email` = '$email'");
+    $user = $result->fetch_assoc();
+
+    if($user != NULL){
+        echo "Пользователь уже есть (Этот логин или пароль уже используеться)";
+        exit();
+    }
+    else{
```
Comment (yurykozhenov) [inline pages/check.php]: No need for `else` after `exit()` in `if`

File: pages/check.php [hunk 16]
```diff
@@ -0,0 +1,85 @@
...
+    if(isset($_POST['accept']) && 
+    $_POST['accept'] == 'true') 
+    {
+        $accept = true;
+
+        if(mb_strlen($login) <= 5 || mb_strlen($login) >= 25){
+            echo "<h1>Логин слишком маленький</h1>";
+            exit();
+        }
+        else{
```
Comment (yurykozhenov) [inline pages/check.php]: No need for `else` after `exit()` in `if`

File: pages/check.php [hunk 17]
```diff
@@ -0,0 +1,85 @@
...
+
+    $country = filter_var(trim($_POST['country']),FILTER_SANITIZE_STRING);
+
+    $city = filter_var(trim($_POST['city']),FILTER_SANITIZE_STRING);
+
+
+    if(isset($_POST['accept']) && 
+    $_POST['accept'] == 'true') 
+    {
+        $accept = true;
```
Comment (yurykozhenov) [inline pages/check.php]: Looks like `$accept` variable is not used anywhere

File: pages/auth.php [hunk 18]
```diff
@@ -0,0 +1,22 @@
...
+    require "../blocks/connect_to_db.php";
+    
+    $result = $mysql->query("SELECT * FROM `users` WHERE `login`= '$login' AND `pass` = '$pass'");
+    $user = $result->fetch_assoc();
+
+    if($user == NULL){
+        echo "Неверный логин или пароль";
+        exit();
+    }
+    else{
```
Comment (yurykozhenov) [inline pages/auth.php:17]: No need for `else` after `exit()`

**Review by yurykozhenov — CHANGES_REQUESTED** [review]

File: blocks/head.php (hunk 1 above)
Comment (tkachuktaras) [reply blocks/head.php]: yes sir

**Review by yurykozhenov — APPROVED** [review]

**Pull request merged**